src/flags/sorting.rs

```diff
@@ -459,10 +433,13 @@ mod test_dir_grouping {
     use super::DirGrouping;
 
     use crate::app;
-    use crate::config_file::Config;
+    use crate::config_file::{Config, Sorting};
     use crate::flags::Configurable;
 
-    use yaml_rust::YamlLoader;
+    #[test]
+    fn test_from_str_bad_value() {
```

Comment:
This needs to be handled differently, if an invalid value will cause a panic or print an error.